sql/dd/impl/bootstrap/bootstrapper.cc

```diff
@@ -1072,6 +1072,27 @@ bool create_dd_schema(THD *thd) {
 }
 
 bool initialize_dd_properties(THD *thd) {
+  if (!opt_initialize) {
+    // Find out which ddse contains dd_properties table during restart
```

Comment:
There can only be one DDSE, so the comment is not correct. I'd just say SE
```suggestion
    // Find out which SE contains dd_properties table during restart
```